### PR TITLE
Alpine support

### PR DESCRIPTION
**Pull request by melezhik**
This is just a draft. For some reasons some of tests fail on SparrowCI - https://ci.sparrowhub.io/report/1488

### CODE REVIEW

**Comment by melezhik** [comment]
```
02:54:29 :: 2022/11/03 02:54:29 [INFO] coreutils not installed
02:54:29 :: 2022/11/03 02:54:29 [ERROR] running /sbin/apk info -qe freedoom: 'exit status 1'
02:54:29 :: 2022/11/03 02:54:29 [INFO] freedoom not installed
02:54:29 :: --- FAIL: TestPkgsToInstall (0.07s)
```

I guess this is because coreutils are not preinstalled on vanilla Alpine Linux

**Comment by melezhik** [comment]
it boils down to this assumption (not met on Alpine) -  https://github.com/ema/pets/blob/master/planner_test.go#L15-L17

```go
  // Test with one package already installed
  pf, err := NewPetsFile("/dev/null", "coreutils", "/etc/passwd", "root", "root", "0640", "", "")
  assertNoError(t, err)
```

**Comment by melezhik** [comment]
```
util.go:79: /bin/chmod 0644 /dev/null != /usr/bin/chmod 0644 /dev/null
```

This is because assumption for location of `chmod` for alpine  is wrong - https://github.com/ema/pets/blob/e3d78d14564745f4f69c58e5642a1fe353557e75/planner_test.go#L112

```go
pa := Chmod(pf)
  if pa != nil {
    t.Errorf("Expecting nil, got %v instead", pa)
  }

  pf.AddMode("0644")

  pa = Chmod(pf)

  assertEquals(t, pa.Cause.String(), "CHMOD")
  assertEquals(t, pa.Command.String(), "/usr/bin/chmod 0644 /dev/null")
```

it's indeed at `/bin/chmod`

File: package.go [hunk 1]
```diff
@@ -126,6 +150,8 @@ func InstallCommand() *exec.Cmd {
 		return NewCmd([]string{"apt-get", "-y", "install"})
 	case YUM:
 		return NewCmd([]string{"yum", "-y", "install"})
+	case APK:
+		return NewCmd([]string{"apk", "add"})
 	}
-	return nil
+  return nil
```
Comment (ema) [inline package.go:156]: Indentation looks off here, please use `go fmt`.

File: package.go [hunk 2]
```diff
@@ -111,10 +126,19 @@ func (pp PetsPackage) IsInstalled() bool {
...
 		}
-
 		return len(stdout) > 0
 	}
 
+	if family == APK {
+		installed := NewCmd([]string{"apk", "info", "-qe", string(pp)})
+		_, _, err := RunCmd(installed)
+		if err != nil {
+			log.Printf("[ERROR] running %s: '%s'", installed, err)
```
Comment (ema) [inline package.go:136]: Missing `\n`.

**Review by ema — CHANGES_REQUESTED** [review]
Two comments, LGTM otherwise! Thanks.

**Comment by melezhik** [comment]
please see now ...

**Review by ema — APPROVED** [review]
Nice, thank you!

**Pull request merged**